### ISSUE TITLE
kb-item, native link tag and Enter key press

### ISSUE THREAD

**Issue by ymaz** [bfanger/angular-keyboard#10]
Hi, I just would like to ask about the following situation:

```html
<ul class="list" kb-list>
  <li class="list-item">
    <a class="list-item-link" href="/some/link" kb-item>Item 1</a>
  </li>
</ul>
```

So, i'm expecting, that when i'm navigate though the list, and select proper link item in menu `<a href="/some/link">` and then press Enter link should start executing as native browser behavior. Instead of this, it's seems like you used `e.preventDefault()` or something.

So, maybe i could achieve (or emulate) this behavior with `<a href="/some/link">` via `kb-invoke` directive?

Thanks.

**Comment by dianafaye17** [comment]
Seconded.

**Comment by jongunter** [comment]
+1 And can we make this work with `ui-sref` links for those of us using angular-ui-router. Thanks!

**Comment by bfanger** [comment]
Fixed in 0.3.2
